containers/ddev-webserver/files/etc/nginx/nginx_typo3.conf

```diff
@@ -38,12 +38,19 @@
     }
 
     # Media: images, icons, video, audio, HTC
-    location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
+    location ~* \.(?:ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
         expires 1M;
         access_log off;
         add_header Cache-Control "public";
     }
 
+    location ~* ^/fileadmin/.+\.(png|jpg|jpeg)$ {
```

Comment:
The `/fileadmin/` prefix looks wrong here, what if a webp is located in a extension e. g. my sitepackage? Also the slashes should be escaped.

Reply:
I'm taking all images that are uploaded by TYPO3 panel user, but of course, this might be changed to automatic match of image files